<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonUtilPage</class>
 <widget class="QWidget" name="JsonUtilPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="selectCsvFilePathButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Select CSV file path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="csvFilePathLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>881</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="convertJsonToCSVButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert Json to CSV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="conversionResultLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>881</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
